### ISSUE TITLE
Run fork mode more frequently (always?) on multi-core machines.

### ISSUE THREAD

**Issue by jonathanmetzman** [google/clusterfuzz#621]
Our Windows bots have 2 cores each.
It looks like our Mac bots have 8.
I think fork mode probably provides a big performance boost on these platforms. We should look into doing it more frequently for them.

This theory could be checked by filtering by platform on the strategy performance graphs.

cc @mbarbella-chromium @mukundv-chrome

**Comment by inferno-chromium** [comment]
Yes we should just enable fork mode by default on multi-cpu machines. And then also fix time_executed to be time_executed * no_of_hours. maybe then change stat from total_fuzzing_time_hrs to cpu_hours.
